src/Components/Playlists/Playlists.jsx

```diff
@@ -1,4 +1,4 @@
-import { useVideo } from "../../Context/reducer";
+import { useVideo } from "../../Context/VideoProvider";
 import "./playlists.css";
 import { NavPane } from "../NavPane/NavPane";
 import { useNavigate } from "react-router-dom";

```

Comment:
On `line 28 - 29` if you want to do `console.log`'s to check the values at the point of time then take this approach instead of what you're doing currently

```suggestion
playlists.map( ( playlist ) => {
    console.log( playlist );
    return (...the JSX you want to return)
})
```